keyboards/pandora/keymaps/all/keymap.c

```diff
@@ -0,0 +1,58 @@
+		KC_TAB,   KC_Q,     KC_W,     KC_E,  KC_R,  KC_T,  KC_Y,  KC_U,   KC_I,  KC_O,     KC_P,     KC_LBRC,  KC_RBRC,  KC_BSLS,
+		KC_CAPS,  KC_A,     KC_S,     KC_D,  KC_F,  KC_G,  KC_H,  KC_J,   KC_K,  KC_L,     KC_SCLN,  KC_QUOT,  KC_NUHS,  KC_ENT,
+		KC_LSFT,  KC_NUBS,  KC_Z,     KC_X,  KC_C,  KC_V,  KC_B,  KC_N,   KC_M,  KC_COMM,  KC_DOT,   KC_SLSH,  KC_RSFT,  MO(1),
+		KC_LCTL,  KC_LGUI,  KC_LALT,                              KC_SPC,                  KC_RALT,  KC_RGUI,  KC_APP,   KC_RCTL),
+
+	LAYOUT_all(
+		KC_GRV,  KC_F1,     KC_F2,    KC_F3,    KC_F4,    KC_F5,    KC_F6,    KC_F7,    KC_F8,    KC_F9,    KC_F10,   KC_F11,   KC_F12,   KC_TRNS,  KC_DEL,
+		KC_TRNS,  RGB_TOG,  RGB_MOD,  RGB_HUI,  RGB_HUD,  RGB_SAI,  RGB_SAD,  RGB_VAI,  RGB_VAD,  KC_TRNS,  KC_PSCR,  KC_SLCK,  KC_PAUS,  RESET,
+		KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_INS,   KC_HOME,  KC_PGUP,  KC_TRNS,  KC_TRNS,
+		KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  BL_DEC,   BL_TOGG,  BL_INC,   BL_STEP,  KC_TRNS,  KC_DEL,   KC_END,   KC_PGDN,  KC_TRNS,  KC_TRNS,
+		KC_TRNS,  KC_TRNS,  KC_TRNS,                                          KC_TRNS,                      KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS),
+
+	LAYOUT_all(
+		KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,
+		KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,
+		KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,
+		KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,
+		KC_TRNS,  KC_TRNS,  KC_TRNS,                                          KC_TRNS,                      KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS),
+
+	LAYOUT_all(
+		KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,
+		KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,
+		KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,
+		KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,
+		KC_TRNS,  KC_TRNS,  KC_TRNS,                                          KC_TRNS,                      KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS)
+};
+
+// Encoder rotate function
+void encoder_update_user(uint8_t index, bool clockwise) {
+  /* First encoder */
+	if (index == 0) {
+    if (clockwise) {
+        tap_code(KC_AUDIO_VOL_UP);
+    } else {
+        tap_code(KC_AUDIO_VOL_DOWN);
+    }
+  }
+}
+
+// Encoder click function
```

Comment:
Is this comment correct, using the dip switches feature for encoder switches?

Reply:
You still havent answered this question, but if the above is true, its kinda a hacky bodge. One that im not fond of landing in the repo

Reply:
Not a hacky dodge, this is how I intended to make the PCB work. Thank you